data_diff/sql.py

```diff
@@ -93,6 +96,9 @@ def compile(self, parent_c: Compiler):
         if self.order_by:
             select += " ORDER BY " + ", ".join(map(c.compile, self.order_by))
 
+        if self.limit is not None:
+            select += " " + c.database.offset_limit(0, self.limit)
```

Comment:
Great you added support for this! Can you change the `test_database_types.py` to use it?